proxy-node-shared/src/test/java/uk/gov/ida/notification/contracts/verifyserviceprovider/TranslatedHubResponseBuilder.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.ida.notification.contracts.verifyserviceprovider;
+
+import java.util.Collections;
+
+import org.joda.time.DateTime;
+
+public class TranslatedHubResponseBuilder {
+
+    public static TranslatedHubResponse buildTranslatedHubResponseAuthenticationFailed() {
+        return new TranslatedHubResponse(VspScenario.AUTHENTICATION_FAILED, "pid1234", VspLevelOfAssurance.LEVEL_2, null);
+    }
+
+    public static TranslatedHubResponse buildTranslatedHubResponseIdentityVerified() {
+        return new TranslatedHubResponse(VspScenario.IDENTITY_VERIFIED, "123456", VspLevelOfAssurance.LEVEL_2, buildAttributes());
+    }
+
+    public static TranslatedHubResponse buildTranslatedHubResponseIdentityVerifiedLOA1() {
+        return new TranslatedHubResponse(VspScenario.IDENTITY_VERIFIED, "123456", VspLevelOfAssurance.LEVEL_1, buildAttributes());
+    }
+
+    public static TranslatedHubResponse buildTranslatedHubResponseIdentityVerifiedNoAttributes() {
+        return new TranslatedHubResponse(VspScenario.IDENTITY_VERIFIED, "123456", VspLevelOfAssurance.LEVEL_2, null);
+    }
+
+    public static TranslatedHubResponse buildTranslatedHubResponseRequestError() {
+        return new TranslatedHubResponse(VspScenario.REQUEST_ERROR, "123456", VspLevelOfAssurance.LEVEL_2, null);
+    }
+
+    public static TranslatedHubResponse buildTranslatedHubResponseOneAttributeOnly() {
```

Comment:
It would be better to rename this method to something like `buildTranslatedHubResponseIncompleteAttributes` to indicate the intention

Reply:
done